pkg/lib/localstore/objectstore/spaceindex/queries.go

```diff
@@ -419,6 +364,37 @@ func (s *dsObjectStore) performFulltextSearch(text string, highlightFormatter ft
 	return results, nil
 }
 
+func convertToHighlightRanges(ranges [][]int, highlight string) []*model.Range {
+	var highlightRanges []*model.Range
+
+	byteToRuneIndex := make([]int, len(highlight)+1)
+	for i := range byteToRuneIndex {
+		byteToRuneIndex[i] = utf8.RuneCountInString(highlight[:i])
```

Comment:
you should work with utf16 (and please, check it in other fts places)

Reply:
I've found in payments

Reply:
it's been reverted in payments